tests/Feature/Posts/UpdatePostsTest.php

```diff
@@ -7,55 +7,47 @@
 use Illuminate\Foundation\Testing\DatabaseTransactions;
 use Illuminate\Foundation\Testing\WithFaker;
 use Illuminate\Http\Response;
+use Illuminate\Support\Carbon;
 use Laravel\Sanctum\Sanctum;
 use Tests\TestCase;
 use Tests\Utility;
 
 class UpdatePostsTest extends TestCase
 {
-    use WithFaker;
-    use DatabaseTransactions;
-
-    private $utility;
-
-    protected function setUp(): void
-    {
-        parent::setUp();
-        $this->utility = new Utility($this);
-        $this->utility->testSetup();
-    }
+    const ALTERED_STRING = 'ALTERED_STRING';
 
     /** @test */
     public function it_does_not_allow_non_auth_users()
     {
-        $post = Post::first();
+        $post = factory(Post::class)->create();
         $this->putJson(route('api.posts.update', $post->id), [
             'name' => $this->faker->name,
             'content' => $this->faker->text,
-            'status_id' => $this->faker->randomElement([Status::DRAFT, Status::PUBLISHED]),
+            'status_id' => $this->faker->randomElement(Status::STATUSES),
         ])->assertStatus(Response::HTTP_UNAUTHORIZED);
     }
 
     /** @test */
     public function it_does_not_allow_guest_to_update_post()
     {
-        $this->utility->loginUser();
-        $post = Post::first();
+        $this->loginUser();
+        $post = factory(Post::class)->create();
         $this->putJson(route('api.posts.update', $post->id), [
             'name' => $this->faker->name,
             'content' => $this->faker->text,
-            'status_id' => $this->faker->randomElement([Status::DRAFT, Status::PUBLISHED]),
+            'status_id' => $this->faker->randomElement(Status::STATUSES),
         ])->assertNotFound();
     }
 
     /** @test */
-    public function it_does_allow_auth_user()
+    public function it_does_allow_auth_user_to_update_post()
     {
-        $this->utility->loginAdmin();
-        $post = Post::first();
-        $post->name = $this->faker->name . "-test";
-        $post->content = $this->faker->text . "-test";
+        $this->loginAdmin();
+        $post = factory(Post::class)->create();
+        $post->name = $this->faker->name . self::ALTERED_STRING;
+        $post->content = $this->faker->text . self::ALTERED_STRING;
         $post->status_id = $post->status_id === Status::DRAFT ? Status::PUBLISHED : Status::DRAFT;
```

Comment:
To reduce any randomness you could set the status_id in your factory call on line 46 then explicitly set what you want it to be updated to